benchmarks/torchbench_model.py

```diff
@@ -69,12 +69,8 @@
     "pytorch_struct": [{
         "test": "eval"
     },],  # not implemented
-    "pytorch_unet": [
-        {
-            # self.load_benchmark() exits the main process. See issue #6207.
-            "xla": "PJRT",
-        },
-    ],
+    # self.load_benchmark() exits the main process. See issue #6207.
+    "pytorch_unet": [{}],
```

Comment:
Does the change mean pytorch_unet can be run on torch_xla?

Reply:
(This is a deny list, not an allow list.)

Reply:
See https://github.com/pytorch/xla/pull/6199 for more details on the problem.